server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/CompactableImpl.java

```diff
@@ -1473,6 +1473,13 @@ public CompactionServiceId getConfiguredService(CompactionKind kind) {
     try {
       var dispatcher = tablet.getTableConfiguration().getCompactionDispatcher();
 
+      if (dispatcher == null) {
```

Comment:
Was this if stmt added to avoid logging an NPE later in the code?

Reply:
Yes, when the context was invalid the logs were being polluted with stack traces.